src/components/global/SideNavigation.tsx

```diff
@@ -45,11 +54,31 @@ export function SideNavigation() {
         <MongoDBLogo height={24} color={darkMode ? "white" : "green-dark-2"} />
       </SideNavItem>
 
-      <SideNavGroup
-        collapsible
-        header="Foundations"
-        glyph={<UniversityIcon />}
-      ></SideNavGroup>
+      {contentPageGroups.map(contentPageGroup => (
```

Comment:
Double check with Alven what counts as a Foundation versus Resource (I think Brand Refresh is resource if I remember correctly.)